docs/build_process.md

```diff
@@ -4,13 +4,25 @@ Here is what the build system is doing under the hood.
 
 ![build structure](assets/build-diagram.png)
 
-1. The build system reads in a configuration
-1. If there is an `includes` attribute in the config, it will take those files and deep merge them into the `properties` object
-1. It takes all the JSON files in the `source` attribute in the config and performs a deep merge onto the `properties` object
-1. Then it iterates over the platforms in the config and:
-  1. Performs all transforms, in order, defined in the transforms attribute or transformGroup
-  1. Builds all files defined in the files array
-  1. Performs any actions defined in the actions attribute
+1. Style Dictionary reads the configuration
+1. Find all property files in the `include` attribute, if it is defined in the config, and deep merge them into the `properties` object. If there are [custom parsers](#parsers) defined, Style Dictionary will run those on files the parsers match.
+1. Find all the property files in the `source` attribute in the config and performs a deep merge onto the `properties` object. If there are [custom parsers](#parsers) defined, Style Dictionary will run those on files the parsers match.
+1. Iterate over the platforms in the config and:
+    1. Perform all transforms, in order, defined in the transforms or transformGroup attribute
+    1. Build all files defined in the files array
+    1. Perform any actions defined in the actions attribute
 
 # How to Build
+
 You can build a style dictionary [using the cli](using_the_cli.md) or [using the npm module](using_the_npm_module.md).
+
+
+## Parsers
+
+Starting in version 3.0, you can define custom parsers to parse property files. This allws you to define your property files in *any* language you like as long as you can write a parser for it. 
```

Comment:
This section needs to be filled out and linked to the properties.md page custom parser section

Reply:
build_process and architecture are 90% the same. Decided to remove build_process, open to renaming architecture build_process though if it makes more sense.